interface/forms/dictation/view.php

```diff
@@ -12,6 +12,8 @@
 
 use OpenEMR\Core\Header;
 include_once("../../globals.php");
+require_once("../../../public/assets/htmlpurifier-4.9.2/library/HTMLPurifier.auto.php");
```

Comment:
The use statement was not needed as @bradymiller suggested. It works without the use statement. I will post in a moment.